src/tribol/common/Parameters.hpp

```diff
@@ -74,12 +74,10 @@ enum VisType
 
 /*!
  * \brief Tribol supports the following different contact modes:
```

Comment:
Might need to update this blurb with the new definition below

Reply:
Updated.